py/desispec/scripts/stdstars.py

```diff
@@ -47,7 +47,7 @@ def parse(options=None):
                          nargs='*',
                          help='List of TARGETIDs of standards overriding the targeting info')
     parser.add_argument('--mpi', action='store_true', help='Use MPI')
-    parser.add_argument('--ignore-gpu', action='store_true', help='Ignore GPU, if available')
+    parser.add_argument('--use-gpu', action='store_true', help='Use GPU, if available')
```

Comment:
desi_extract_spectra uses `--gpu`, I don't have a strong preference between `--use-gpu` or `--gpu` but it's probably good to be consistent.

https://github.com/desihub/desispec/blob/0eede4e36b488cbbc560f90f94d4a37ac3f7f543/py/desispec/scripts/extract.py#L69

Reply:
Thanks for catching this, have changed `--gpu` to `--use-gpu` for desi_extract_spectra.